packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -308,7 +308,7 @@ HTTP/1.1 204 No Content
 
 ## Specify the User Type of a new user
 
-The [Create User](/docs/reference/api/users/#create-user-with-non-default-user-type) operation accepts a type specification as part of the request body. The specification is a map, but currently the only key permitted is `id`. The type specification is also added to the [User object](/docs/reference/api/users/#user-object), but after user creation the type may only be updated by an administrator on a [full replace of an existing user](/docs/reference/api/user-types/#update-user-type), not a partial update.
+The [Create User](/docs/reference/api/users/#create-user-with-non-default-user-type) operation accepts a type specification as part of the request body. The specification is a map, but currently the only key permitted is `id`. The type specification is also added to the [User object](/docs/reference/api/users/#user-object), but after user creation the type may only be updated by an administrator on a [full replace of an existing user](/docs/reference/api/users/#update-user), not a partial update.
```

Comment:
The link works, but I'm not clear on **why** you're linking away from the Update User Type description. The previous destination link (Update User Type) explains that that a PUT does a full replace operation on the user type; the new destination link (Update User) isn't clear up front about how to do that full replace operation to make that change - in fact, the note at the bottom sends them right back to the "full replacement PUT operation" on the Update User Type. I think you need to add some text to explain exactly what the dev needs to do.

Reply:
The existing link to the Update User Type description is not correct. That's where you would change things about the User Type itself not the User Type for a specific user. You are correct that the link in the bottom of the user update section is also incorrect, so thank you for pointing that out, and I've removed the link there. I'm not actually part of the UD team. I just want to fix the mistake in the docs quickly. Could we at least fix the broken links here with my PR, and file a ticket with the ud team, if you feel like there is additional work to be done on the description?

Reply:
OK - can you open the JIRA against the UD team to fix please? I will approve these fixes.

Reply:
I'm actually not sure what I would change about the existing description, so I don't actually have anything to file against the UD team. If you have any specific suggestions though you can follow up with them.